src/Command/General/AttachmentsImagesSubsizesMigrator.php

```diff
@@ -0,0 +1,247 @@
+<?php
+/**
+ * Command for fixing attachments with filenames that are incompatible with Subsizes.
+ *
+ * @package newspack-custom-content-converter
+ */
+
+namespace NewspackCustomContentMigrator\Command\General;
+
+use \NewspackCustomContentMigrator\Command\InterfaceCommand;
+use \NewspackCustomContentMigrator\Logic\Posts;
+use \WP_CLI;
+
+/**
+ * Fixes for attachments with filenames that are incompatible with Subsizes.
+ */
+class AttachmentsImagesSubsizesMigrator implements InterfaceCommand {
+
+	/**
+	 * Instance.
+	 *
+	 * @var null|InterfaceCommand Instance.
+	 */
+	private static $instance = null;
+
+	/**
+	 * Posts instance.
+	 *
+	 * @var Posts Posts instance.
+	 */
+	private $posts;
+
+	/**
+	 * Constructor.
+	 */
+	private function __construct() {
+		$this->posts = new Posts();
+	}
+
+	/**
+	 * Singleton get_instance().
+	 *
+	 * @return InterfaceCommand|null
+	 */
+	public static function get_instance() {
```

Comment:
This is a nit, but there is no reason to keep an instance var for the simpleton, so this would be simpler:
```suggestion
	public static function get_instance(): self {
		static $instance = null;
		if ( null === $instance ) {
			$instance = new self();
		}

		return $instance;
	}
```

Reply:
Updated.